### PR TITLE
Flatten TaskStats and PipelineStats

### PR DESCRIPTION
**Pull request by ajaygeorge**
Flatten TaskStats and PipelineStats

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by ajaygeorge** [comment]
Jenkins is passing.
Will wait for a clean verifier run before merging.

**Comment by ajaygeorge** [comment]
Pushed out a UI change as well which I identified while testing.

File: presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java [hunk 1]
```diff
@@ -153,34 +151,34 @@ public PipelineStats(
         checkArgument(completedDrivers >= 0, "completedDrivers is negative");
         this.completedDrivers = completedDrivers;
 
-        this.userMemoryReservation = requireNonNull(userMemoryReservation, "userMemoryReservation is null");
-        this.revocableMemoryReservation = requireNonNull(revocableMemoryReservation, "revocableMemoryReservation is null");
-        this.systemMemoryReservation = requireNonNull(systemMemoryReservation, "systemMemoryReservation is null");
+        this.userMemoryReservationInBytes = requireNonNull(userMemoryReservationInBytes, "userMemoryReservationInBytes is null");
```
Comment (cemcayiroglu) [inline presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java:154]: primitives can not be null. no need to check.

Comment (ajaygeorge) [reply presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java]: done

**Comment by ajaygeorge** [comment]
@arhimondr I think this PR is looking good to be merged now.

**Pull request merged**